server.js

```diff
@@ -1,9 +1,13 @@
 import config from './config/index.js';
+import LoggerService from './config/logger/logger-service.js';
+
+const logger = new LoggerService('server');
+
    
 export default (app) => {
     const port = config.port;
     
     return app.listen(port, () =>
-            console.log(`Server is running on http://locahost:${port}`));
+        logger.info(`Server is running on http://locahost:${port}`));
```

Comment:
what will it log if you run the code on dev/staging/prod env?

Reply:
fixed

Reply:
fixed #2, now it shows IP